Python/pythonrun.c

```diff
@@ -188,6 +188,24 @@ PyRun_InteractiveLoopFlags(FILE *fp, const char *filename, PyCompilerFlags *flag
 
 }
 
+static int
+PyRun_IsInteractiveExitCommand(mod_ty mod) {
+    if (asdl_seq_LEN(mod->v.Module.body) != 1) {
+        return 0;
+    }
+    stmt_ty statement = asdl_seq_GET(mod->v.Module.body, 0);
+    if (statement->kind != Expr_kind) {
+        return 0;
+    }
+    expr_ty expr = statement->v.Expr.value;
+    if (expr->kind != Name_kind) {
+        return 0;
+    }
+    if (expr->v.Name.ctx != Load) {
+        return 0;
+    }
+    return PyUnicode_CompareWithASCIIString(expr->v.Name.id, "exit") == 0;
```

Comment:
The ``quit()`` people feel a little lonely here 😉 .

Reply:
If we decide to go with this, I will finish the PR 😉  (There are some other details missing)

Reply:
Right. In any case, thanks for taking this issue up! I don't think there's such a precedent for special repl-only parsing, so I'm impressed to see a working implementation.